### PR TITLE
Angus branch to master

### PR DESCRIPTION
**Pull request by nwarmenh**

### CODE REVIEW

File: connection.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+//$db = new mysqli('localhost:8889', 'root', 'root', 'attendance_new');
+	$db = new mysqli('sql3.freemysqlhosting.net:3306', 'sql3218514', 'fxBw214YQ4', 'sql3218514');
```
Comment (badtzmaru) [inline connection.php]: You might not want this in a public repo 🤔

Comment (nwarmenh) [reply connection.php]: Geez, I'm catching hell for this in Orcas right now too. Who cares for now? It's just for testing.

Comment (nwarmenh) [reply connection.php]: Better than everyone swapping DB files to their local AMPs

Comment (badtzmaru) [reply connection.php]: Well I guess it's fine as long as you have a backup. People scrape github looking for this kinda thing so your tables could probably get dropped.

Comment (badtzmaru) [reply connection.php]: Swapping dbs around is bad, but you could just use a gitignore to remove the connection file.

Comment (pscsstudentsam) [reply connection.php]: Oh Milo... 😍

Comment (henry956) [reply connection.php]: Jack!!!!!!!!

Comment (henry956) [reply connection.php]: A comment

**Pull request merged**